include/bio/var_io/header.hpp

```diff
@@ -214,6 +218,19 @@ class header
     /*!\name Update, reset and inspect
      * \{
      */
+    /*! \brief Gets the current time and transforms it in a nice readable way for the vcf header line fileDate.
+     *
+     * \returns a time string in the format: YYYY-MM-DD HH:MM:SS.
+     */
+    std::string transTime()
```

Comment:
This function seems a little bit out-of-place here, because it is a generic "make a nice date"-function and not specific 

I am also not sure what the name implies? 🏳️‍⚧️ 🕐 ? 😉

Reply:
Oh that was just a name copy of some funktions I found in the wild wild web.. 🙈 If we ceep the function, we can of course rename it to transformTime for example.